filebeat/module/kibana/_meta/fields.yml

```diff
@@ -6,4 +6,41 @@
     - name: kibana
       type: group
       description: >
+        Module for parsing Kibana logs.
       fields:
+        - name: space_id
+          description: "The id of the space associated with this request."
```

Comment:
I would say "associated with this event" instead of "request" since it could also be a background task.